Composer/packages/server/src/models/bot/botProject.ts

```diff
@@ -205,12 +205,22 @@ export class BotProject implements IBotProject {
     // Resolve relative path for custom runtime if the path is relative
     if (settings?.runtime?.customRuntime && settings.runtime.path && !Path.isAbsolute(settings.runtime.path)) {
       const absolutePath = Path.resolve(this.dir, 'settings', settings.runtime.path);
+
       if (fs.existsSync(absolutePath)) {
         settings.runtime.path = absolutePath;
         await this.updateEnvSettings(settings);
       }
     }
 
+    if (
```

Comment:
is this not something that can be done inside the yeoman template?

Reply:
Good point, I just validated it and it works nicely

Reply:
Kinda makes you think how much of the flow should/could be handle by the template itself. Things like the build command could be templated as well but that may be overkill
![image](https://user-images.githubusercontent.com/22845013/104507153-33115d80-559b-11eb-9512-2ee730f180cf.png)
In a larger sense the template itself could probably run things like dialog merge once it completes file generation but may be best to keep that logic in Composer if we are not needing to optimize on non Composer use of this

Reply:
Yeah I guess im just realizing the power of Yeomen, cool stuff!